source/declarations.tex

```diff
@@ -1954,6 +1954,20 @@
 that contains a placeholder type,
 \tcode{T} is the declared type of the non-type template parameter
 and $E$ is the corresponding template argument.
+\item
```

Comment:
I think this would make more sense as the second or third bullet, so that it's immediately adjacent to the very similar variable case.

Reply:
(I think that would make sense.)

Reply:
Yes, I see it now --  I confused myself among the multiple nested lists. Done!